bootstrap-config/data-workspace-apps-copilot.yml

```diff
@@ -55,47 +54,8 @@ services:
       url: countries-of-interest-service-staging.london.cloudapps.digital
   image_location: public.ecr.aws/uktrade/copilot-bootstrap:latest
   name: countries-of-interest-service
-  overlapping_secrets:
-  - FLASK_ENV
   repo: git@github.com:uktrade/countries-of-interest-service.git
-  secrets:
-    ABC_CLIENT_ID: ABC_CLIENT_ID
-    ABC_CLIENT_SECRET: ABC_CLIENT_SECRET
-    AC_HAWK_ENABLED: AC_HAWK_ENABLED
-    CMS_BASE_URL: CMS_BASE_URL
-    COUNTRIES_AND_TERRITORIES_GROUP_SLUG: COUNTRIES_AND_TERRITORIES_GROUP_SLUG
-    COUNTRIES_AND_TERRITORIES_REFERENCE_SLUG: COUNTRIES_AND_TERRITORIES_REFERENCE_SLUG
-    DATAFLOW_HAWK_CLIENT_ID: DATAFLOW_HAWK_CLIENT_ID
-    DATAFLOW_HAWK_CLIENT_KEY: DATAFLOW_HAWK_CLIENT_KEY
-    DATAHUB_COMPANIES_DATASET_ID: DATAHUB_COMPANIES_DATASET_ID
-    DATAHUB_COMPANIES_SOURCE_TABLE_ID: DATAHUB_COMPANIES_SOURCE_TABLE_ID
-    DATAHUB_COMPANY_EXPORT_COUNTRY_DATASET_ID: DATAHUB_COMPANY_EXPORT_COUNTRY_DATASET_ID
-    DATAHUB_COMPANY_EXPORT_COUNTRY_HISTORY_DATASET_ID: DATAHUB_COMPANY_EXPORT_COUNTRY_HISTORY_DATASET_ID
-    DATAHUB_COMPANY_EXPORT_COUNTRY_HISTORY_SOURCE_TABLE_ID: DATAHUB_COMPANY_EXPORT_COUNTRY_HISTORY_SOURCE_TABLE_ID
-    DATAHUB_COMPANY_EXPORT_COUNTRY_SOURCE_TABLE_ID: DATAHUB_COMPANY_EXPORT_COUNTRY_SOURCE_TABLE_ID
-    DATAHUB_CONTACTS_DATASET_ID: DATAHUB_CONTACTS_DATASET_ID
-    DATAHUB_CONTACTS_SOURCE_TABLE_ID: DATAHUB_CONTACTS_SOURCE_TABLE_ID
-    DATAHUB_EXPORT_COUNTRIES_DATASET_ID: DATAHUB_EXPORT_COUNTRIES_DATASET_ID
-    DATAHUB_EXPORT_COUNTRIES_SOURCE_TABLE_ID: DATAHUB_EXPORT_COUNTRIES_SOURCE_TABLE_ID
-    DATAHUB_EXPORT_COUNTRY_HISTORY_DATASET_ID: DATAHUB_EXPORT_COUNTRY_HISTORY_DATASET_ID
-    DATAHUB_EXPORT_COUNTRY_HISTORY_SOURCE_TABLE_ID: DATAHUB_EXPORT_COUNTRY_HISTORY_SOURCE_TABLE_ID
-    DATAHUB_FUTURE_INTEREST_COUNTRIES_DATASET_ID: DATAHUB_FUTURE_INTEREST_COUNTRIES_DATASET_ID
-    DATAHUB_FUTURE_INTEREST_COUNTRIES_SOURCE_TABLE_ID: DATAHUB_FUTURE_INTEREST_COUNTRIES_SOURCE_TABLE_ID
-    DATAHUB_INTERACTIONS_DATASET_ID: DATAHUB_INTERACTIONS_DATASET_ID
-    DATAHUB_INTERACTIONS_EXPORT_COUNTRY_DATASET_ID: DATAHUB_INTERACTIONS_EXPORT_COUNTRY_DATASET_ID
-    DATAHUB_INTERACTIONS_EXPORT_COUNTRY_SOURCE_TABLE_ID: DATAHUB_INTERACTIONS_EXPORT_COUNTRY_SOURCE_TABLE_ID
-    DATAHUB_INTERACTIONS_SOURCE_TABLE_ID: DATAHUB_INTERACTIONS_SOURCE_TABLE_ID
-    DATAHUB_OMIS_DATASET_ID: DATAHUB_OMIS_DATASET_ID
-    DATAHUB_OMIS_SOURCE_TABLE_ID: DATAHUB_OMIS_SOURCE_TABLE_ID
-    DATAWORKSPACE_BASE_URL: DATAWORKSPACE_BASE_URL
-    EXPORT_WINS_DATASET_ID: EXPORT_WINS_DATASET_ID
-    EXPORT_WINS_SOURCE_TABLE_ID: EXPORT_WINS_SOURCE_TABLE_ID
-    FLASK_ENV: FLASK_ENV
-    HAWK_CLIENT_ID: HAWK_CLIENT_ID
-    HAWK_CLIENT_KEY: HAWK_CLIENT_KEY
-    PAGINATION_SIZE: PAGINATION_SIZE
-    RUN_SCHEDULER: RUN_SCHEDULER
-    STUB_SOURCE_DATA: STUB_SOURCE_DATA
+  secrets: {}
```

Comment:
I'm wondering why quite a few config files are losing their secrets?  🤔

Reply:
Good catch. Updated the `build-copilot-config.py` script accordingly in https://github.com/uktrade/copilot-tools/pull/17/commits/91e0a56b5f458479b6ebee36de430ee34bd792b5.